l1-contracts/src/core/Rollup.sol

```diff
@@ -395,10 +391,10 @@ contract Rollup is Leonidas, IRollup, ITestRollup {
       revert Errors.Rollup__InvalidArchive(tipArchive, _archive);
     }
 
-    address proposer = getProposerAt(_ts);
-    if (proposer != address(0) && proposer != _proposer) {
-      revert Errors.Leonidas__InvalidProposer(proposer, _proposer);
-    }
+    SignatureLib.Signature[] memory sigs = new SignatureLib.Signature[](0);
+    DataStructures.ExecutionFlags memory flags =
+      DataStructures.ExecutionFlags({ignoreDA: true, ignoreSignatures: true});
+    _proposePendingBlock(slot, sigs, _archive, flags);
```

Comment:
_proposePendingBlock is a confusing function name, from the def it is just performing checks, not actually doing the proposing?